monGARS/core/aui.py

```
@@ -0,0 +1,115 @@
+
+        if self._embed:
+            try:
+                prompt_vector = self._embed.encode([prompt])[0]
+                action_vectors = self._embed.encode(
+                    [description for _, description in items]
+                )
+                return {
+                    key: _cosine(prompt_vector, vector)
+                    for (key, _), vector in zip(items, action_vectors, strict=False)
+                }
+            except Exception as exc:  # pragma: no cover - runtime fallback
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)